### PR TITLE
add function for connection health

### PR DESCRIPTION
**Pull request by KPostOffice**
## This introduces a breaking change

- [ ] Yes
- [x] No

## Has a Message Class been changed? Version incremented?

- [ ] Yes
- [x] No

## This should yield a new module release

- [ ] Yes
- [ ] No

## This Pull Request implements

This function allows a user to check connection to Kafka.  There is no direct way to do this so the easiest thing to do is create a query which simply asks for info about current topics. If this does not complete in the given amount of time then `False` otherwise `True`

### CODE REVIEW

File: thoth/messaging/admin_client.py [hunk 1]
```diff
@@ -68,3 +68,17 @@ def create_topic(admin: AdminClient, message: MessageBase, partitions: int = 1,
...
+    if config:
+        a = AdminClient(config)
+    else:
+        a = AdminClient(kafka_config_from_env())
+
+    try:
+        a.list_topics(timeout=timeout)
+        return True
+    except Exception:
+        return False
```
Comment (fridex) [inline thoth/messaging/admin_client.py:84]: Would it make sense to report the exception here to get information in logs for further debugging?

**Review by fridex — COMMENTED** [review]
One minor question, otherwise LGTM! :100:

File: thoth/messaging/admin_client.py (hunk 1 above)
Comment (goern) [reply thoth/messaging/admin_client.py:84]: +1

Comment (KPostOffice) [reply thoth/messaging/admin_client.py:84]: :+1:

**Comment by KPostOffice** [comment]
@fridex PTAL

**Review by fridex — APPROVED** [review]
lgtm, thanks! :100: :+1:

**Pull request merged**